homework-submission/react-hyf/react-hyf-client/src/React_HW2/todoApp2.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+
+// ContentDynamicTodo component is for map function to use
+const ContentDynamicTodo = ({ description, deadline, className, deleteItem }) => {
+  return (
+    <tbody>
+      <tr className={className} onClick={deleteItem}>
+        <td class="evenBackgroundColor">
+          <td class="content"> {description} </td> <td class="content"> {deadline} </td>{' '}
+          <td class="content"> {className} </td>{' '}
+        </td>{' '}
+      </tr>{' '}
+    </tbody>
+  );
+};
+
+// DynamicList where map function is
+const DynamicList = ({ list, deleteItem }) => {
+  const runTodoList = list.map(todo => (
+    <ContentDynamicTodo
+      key={todo.id}
```

Comment:
Please have a look to the previous homework review.

Reply:
I push the updated one for HW2

Reply:
And implement it in HW3 also.